src/accumulator/util.rs

```diff
@@ -12,27 +12,62 @@ pub fn is_root_position(position: u64, num_leaves: u64, forest_rows: u8) -> bool
     return root_present && root_pos == position;
 }
 
-// extractTwins is a optimization for batched deletions. It checks if the nodes
-// being deleted also have their sibling being deleted. It returns the parents
-// of the deleted siblings along with nodes that didn't have a sibling
-pub fn extract_twins(nodes: Vec<u64>, forest_rows: u8) -> (Vec<u64>, Vec<u64>) {
-    let mut parents = Vec::new();
-    let mut twined = Vec::new();
+// removeBit removes the nth bit from the val passed in. For example, if the 2nd
```

Comment:
Could we move these util functions into a different commit or a new PR? Would be very helpful for reviewing.